### PR TITLE
added a contributing.md

### LINKED ISSUE #5: Add Contribution.md
Tell open devs how can they contribute

### PR DESCRIPTION
**Pull request by Kanchan253**
Hi, I have added a contributing.md file which has the steps on how to contribute to the project. Please check and review it and let me know if I have to make any changes in it. Thank you!

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+●Click the "Fork" button at the top right corner of this repository's page.
+
+![Alt text](../../Users/Lenovo/Documents/fork.png)
+
+## Step 2:Clone the forked repository
+
+●Go to your GitHub account and open the forked repository.
+●Click the "Code" button and then the copy icon to copy the repository's URL.
+
+![Alt text](../../Users/Lenovo/Documents/code.png)
```
Comment (silenteyesoncode) [inline Contributing.md]: It appears that there is an issue with the image URL provided in the code snippet. The path `"../../Users/Lenovo/Documents/fork.png"` suggests a local file path on your personal computer, which cannot be accessed or displayed on the internet.

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+![Alt text](../../Users/Lenovo/Documents/copy.png)
+
+The command will look something like this :
+git clone https://github.com/{your github user name}/AI-Powered-News-Article-Classifier-with-History.git
+
+## Step 3:Navigate to the Repository
+
+●Open the terminal and change your directory to the cloned repository.
+
+cd AI-Powered-News-Article-Classifier-with-History/
```
Comment (silenteyesoncode) [inline Contributing.md]: Can we have to different path command for backend and frontend folders

**Review by silenteyesoncode — CHANGES_REQUESTED** [review]

File: Contributing.md (hunk 1 above)
Comment (Kanchan253) [reply Contributing.md]: Should I add the images to a file in the project?

File: Contributing.md (hunk 2 above)
Comment (Kanchan253) [reply Contributing.md]: So I have to add a separate path for the backend folder for the people who want to contribute to the backend and the same for the frontend part of it?

**Review by silenteyesoncode — CHANGES_REQUESTED** [review]
Hi @Kanchan253 

I Appreciate the efforts here, although with respect to open source repository standers i may need some changes here 
you can take reference from  [OpenMind](https://github.com/The-Unleashed-Club/openMind/blob/main/Contributing.md) repo 

###  [Feel free to react out to me for any query ](https://calendly.com/7silenteyes07/professional-coffee-chat)

File: Contributing.md (hunk 1 above)
Comment (silenteyesoncode) [reply Contributing.md]: yes you can upload the image in GitHub itself here

File: Contributing.md (hunk 2 above)
Comment (silenteyesoncode) [reply Contributing.md]: Yes, U got me

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## 3.Changes to the Code:
+
+If you're interested in making changes to the project's code, please do so. In order to submit your code changes, follow these steps:
+
+### Step 1: Fork this repository
+
+●Click the "Fork" button at the top right corner of this repository's page.
+
+![Alt text](images/fork.png)
```
Comment (silenteyesoncode) [inline Contributing.md]: can you please remove Alt text
and add ``

Comment (silenteyesoncode) [reply Contributing.md]: If you give me the edit access I can show you
@Kanchan253

Comment (Kanchan253) [reply Contributing.md]: @silenteyesoncode I have given you access.

Comment (silenteyesoncode) [reply Contributing.md]: great i'll made some changes and Merge the PR By Monday

Comment (Kanchan253) [reply Contributing.md]: Ok sure

Comment (silenteyesoncode) [reply Contributing.md]: Thanks you so much for your contribution. I'll show your profile in in repo readme for sure 
Wish you a good luck

Comment (Kanchan253) [reply Contributing.md]: Thank you!!

**Pull request merged**